template/login.html

```diff
@@ -19,48 +19,88 @@
-            ><br />
-            <input type="text" id="loginusername" name="username" required
-            /><br /><br />
-            <label for="password"><b>Password:</b></label><br />
-            <input type="password" id="loginpword" name="password" required /><br />
-            <input type="checkbox" onclick="showPass()" />Show Password
-            <br />
-            <div class="form-group">
-        <a href="password_reset.html">Forgot Password?</a> <!-- Link to the password reset page -->
-      </div>
-            <div class="form-group checkbox">
-        <input type="checkbox" id="terms" name="terms" required>
-        <label for="terms">I agree to the terms and conditions</label>
-      </div>
-
-            <input type="submit" value="Login" id="loginsubBtn" /><br /><br />
-
-          </form>
-          <a href="{% url 'acctype' %}"><input type="button" value="Create New Account" id="createNewAccBtn" /></a>
 
+      <section class="vh-100" style="background-color: #9A616D;">
+        <div class="container py-5 h-100">
+          <div class="row d-flex justify-content-center align-items-center h-100">
+            <div class="col col-xl-10">
+              <div class="card" style="border-radius: 1rem;">
+                <div class="row g-0">
+                  <div class="col-md-6 col-lg-5 d-none d-md-block">
+                    <img src="https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-login-form/img1.webp"
+                      alt="login form" class="img-fluid h-100" style="border-radius: 1rem 0 0 1rem;" />
+                  </div>
+                  <div class="col-md-6 col-lg-7 d-flex align-items-center">
+                    <div class="card-body p-4 p-lg-5 text-black">
+                      <div>
+                      {% for message in messages %}
+                      <div class="alert alert-warning alert-dismissible fade show" role="alert">
+                        <strong>Alert &nbsp;</strong>{{message}}
+              
+                      </div>
+                      {% endfor %}
+                    </div>
+                    <div>
+                    {% for message in messages %}
+                    <div class="alert alert-warning alert-dismissible fade show" role="alert">
+                      <strong>Alert</strong>{{message}}
+            
+                    </div>
+                    {% endfor %}
+                  </div>
+          
+            
+            
+            
+                
+              
+                      <form action="" method="POST">
+                        {% csrf_token %}
+                        <div class="d-flex align-items-center mb-3 pb-1">
+                          <span class="h1 fw-bold mb-0">WorkPlace</span>
+                        </div>
+                        <div class="d-flex justify-content-between">
+                        <h5 class="fw-normal text-black" style="letter-spacing: 1px;">Sign into account</h5>
```

Comment:
felt it would be better if there is space.
``Sign in to account``

And also can you add padding below this text, it is kind of looking conjusted now.

Reply:
<img width="495" alt="Screenshot 2024-05-09 at 3 56 34 PM" src="https://github.com/nothuman2718/WorkPlace/assets/133094641/38f548a9-2607-4632-ae33-d5d482c6f333">

Reply:
ok sure will add it